client/state/sites/schema.js

```diff
@@ -57,7 +57,8 @@ export const sitesSchema = {
 						is_free: { type: 'boolean' }
 					}
 				},
-				single_user_site: { type: 'boolean' }
+				single_user_site: { type: 'boolean' },
+				updates: { type: 'object' },
```

Comment:
To clarify, unless we specify `additionalProperties: false` in the schema, additional unexpected properties are tolerated during the schema check. So in this case if we actually added this, it'd need to be `updates: type:[ 'object', 'null' ]` otherwise we'd throw out any persisted site data for normal wpcom sites.